CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# dbt_qualtrics v0.2.0
```

Comment:
Source Changelog has v.Number.Number in front of the package, transform has v.Number.Number behind it. I know the auto-releaser will usually take care of this but since it's broken, something to watch out for in manual release.

Reply:
Good call, I just updated the ordering to be consistent with the source and our other packages.